### PR TITLE
Add comments page

### PR DESCRIPTION
**Pull request by erichli1**
Add comments and draw functionality. Also, load images in my work after click.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -26,28 +33,99 @@
...
+  private static final String REQUEST_PARAMETER_PICTURE_LINK = "picture-link";
+  private static final String REQUEST_PARAMETER_NUMBER_COMMENTS = "number-comments";
+  private static final String REDIRECT_COMMENTS = "/comments.html";
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<Comment> comments = new ArrayList<>();
+
+    int numberComments = 0;
+    int maxNumberComments = getPositiveInt(request);
```
Comment (hdhabu) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:46]: Does getPositiveInt return -1 in the failed case? Hence the protection on Lines 48-50?

Can the returned value be 0? Line 66 might fail forever then, right?

Comment (hdhabu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:46]: Okay, saw the code below. 
-1 is error case, yes. 
0 cannot be returned.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -295,7 +280,7 @@ <h1 class="title">Contact me</h1>
                 <a href="https://github.com/erichli1" target="_blank" style="text-decoration: none;">
                     <i class="fab fa-github fa-2x"></i>
                 </a>
-            </div>
+            </div>
```
Comment (hdhabu) [inline portfolio/src/main/webapp/index.html]: Extra whitespace at the end?

**Review by hdhabu — COMMENTED** [review]
Just one whitespace issue which needs addressing

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (erichli1) [reply portfolio/src/main/webapp/index.html]: Just committed a change to remove the whitespace!

**Review by hdhabu — APPROVED** [review]
LGTM

**Pull request merged**